aleph-client/src/system.rs

```diff
@@ -19,3 +19,19 @@ pub fn set_code(connection: &RootConnection, runtime: Vec<u8>, status: XtStatus)
     );
     send_xt(connection, xt, Some("set_code"), status);
 }
+
+impl CallSystem for RootConnection {
+    type Error = substrate_api_client::error::Error;
+
+    fn fill_block(&self, target_ratio: u32, status: XtStatus) -> Result<(), Self::Error> {
+        let connection = self.as_connection();
+        let call = compose_call!(
+            connection.metadata,
+            "System",
+            "fill_block",
+            target_ratio * 10_000_000
```

Comment:
Request: name this `10_000_000` constant - perhaps it's obvious to someone who's been in the project for some time, but I have no clue why it has the value it does.

Reply:
Actually, I have no clue as well. I will ask the author and introduce some descriptive name.

Reply:
Changed - now it should be more descriptive.

Reply:
It still says `10_000_000` here, did you forget to push something?

Reply:
Check it now.

Reply:
Can we use https://docs.rs/sp-runtime/6.0.0/sp_runtime/struct.Perbill.html#method.from_percent here? Or is this a different perbill?

Reply:
Changed it.